csharp/ql/src/Documentation/XmldocExtraTypeParam.ql

```diff
@@ -1,6 +1,6 @@
 /**
  * @name Incorrect type parameter name in documentation
- * @description The type parameter name given in a '<typeparam>' tag does not exist. Rename the parameter or
+ * @description The type parameter name given in a `<typeparam>` tag does not exist. Rename the parameter or
```

Comment:
If this is the new way of doing things, we should update the comment `Any code elements included in the description should be enclosed in single quotes` (https://github.com/github/codeql/blob/main/docs/query-metadata-style-guide.md?plain=1#L63), as well as the example.

Reply:
I should clarify that this is not a style change but a rendering fix.  Here is what the documentation currently looks like:

https://codeql.github.com/codeql-standard-libraries/csharp/Documentation/XmldocExtraTypeParam.ql/module.XmldocExtraTypeParam.html
<img width="600" alt="typeparam" src="https://user-images.githubusercontent.com/1418580/165976162-5555b9d3-b7ca-4b3c-b7bf-d114b30cc282.png">

QL comments can include Markdown and HTML, so the `<` and `>` characters need to be escaped, either by putting them in Markdown code format (the approach I took here) or by replacing them with `&lt;` and `&gt;`.  I can switch it to the second approach if that is more style-compliant.

I agree that using Markdown backquotes for code elements seems sensible, but that should probably be a separate discussion.

Reply:
> this is not a style change but a rendering fix

No, it's _both_ a style change and a rendering fix.